### PR TITLE
Velocity Migration from 1.7 to 2.0

### PR DESCRIPTION
**Pull request by amielkelinkedin**
Tested on solo server and via two internal servers - for reportal and azkaban functionality.

### CODE REVIEW

**Review by djaiswal83 — APPROVED** [review]
Looks good to me. Thanks for cleaning up tech debt.

File: az-reportal/src/main/java/azkaban/viewer/reportal/ReportalServlet.java [hunk 1]
```diff
@@ -398,14 +398,14 @@ private void handleViewReportal(final HttpServletRequest req,
         }
         // View single log
         if (hasParam(req, "log")) {
-          page.add("view-log", true);
+          page.add("viewLog", true);
```
Comment (svnarumugam) [inline az-reportal/src/main/java/azkaban/viewer/reportal/ReportalServlet.java:401]: Clarification: I am curious to know if this change and a few other changes for view-logs, view-preview, view-executions are mandated by the library or we are doing this to keep it clean. Thanks a lot for picking up this change.

**Review by svnarumugam — COMMENTED** [review]
Thank you for working on this upgrade! Added a clarifying question. rest looks good to me.

File: az-reportal/src/main/java/azkaban/viewer/reportal/ReportalServlet.java (hunk 1 above)
Comment (amielkelinkedin) [reply az-reportal/src/main/java/azkaban/viewer/reportal/ReportalServlet.java:401]: This is mandated by the library. Hyphens are no longer allowed. I believe underscores are allowed (_) if we would like to change from view-log to view_log.

Comment (svnarumugam) [reply az-reportal/src/main/java/azkaban/viewer/reportal/ReportalServlet.java:401]: Thanks a lot @amielkelinkedin ! :)

**Review by svnarumugam — DISMISSED** [review]
Thanks for working on this change Adam.

**Review by byjiang1996 — DISMISSED** [review]
LGTM. Thanks for fixing it!!!

**Review by byjiang1996 — APPROVED** [review]

**Pull request merged**